src/knot_protocol.c

```diff
@@ -131,6 +131,37 @@ int knot_value_type_is_valid(uint8_t type)
 	return KNOT_ERR_INVALID;
 }
 
+int knot_value_assign_limit(int value_type, knot_value_type value,
+			    knot_value_type *limit)
+{
+	switch (value_type) {
+	case KNOT_VALUE_TYPE_INT:
+		limit->val_i = value.val_i;
+		break;
+	case KNOT_VALUE_TYPE_FLOAT:
+		limit->val_f = value.val_f;
+		break;
+	case KNOT_VALUE_TYPE_BOOL:
+		limit->val_b = value.val_b;
```

Comment:
The KNOT_VALUE_TYPE_BOOL has no limits according to [knot_config_is_valid](https://github.com/CESARBR/knot-protocol-source/blob/c7af200d735abf2e02fdae6eb6c885dc750e0d7c/src/knot_protocol.c#L229) function.

Reply:
Maybe it should be removed from this function or from knot_config_is_valid function.

Reply:
I think it's better to not assign a limit to a boolean in this PR. If its necessary, we can add this change later, but I don't think it is necessary right now.

Reply:
Ok, i agree.
Done